data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<p>I like how you improved the friendliness of the output over the previous iteration. At the same time, I thought the previous diagnostic message was useful-- this has a little less information, and it's generally better to over-communicate than under-communicate. Would you mind adding the grouping and partitioning strategies here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>